hook

```diff
@@ -38,6 +38,11 @@ elif [[ -x "$LOCAL" ]]; then
 fi
 
 #
+# Figure out the location of the pre-commit package
+#
+PRECOMMIT="$(dirname "$(readlink "$0")")"
```

Comment:
This line doesn't work on window `.git/hooks/pre-commit: line 43: readlink: command not found`